fixtures/pages-workerjs-directory/public/_worker.js/index.js

```diff
@@ -15,11 +15,31 @@ export default {
 		}
 
 		if (pathname === "/d1") {
-			const stmt = env.D1.prepare("SELECT 1");
+			const stmt1 = env.D1.prepare("SELECT 1");
+			const values1 = await stmt1.first();
```

Comment:
We were previously asserting the output of this command, but not anymore. Are we okay with that?

Also maybe we should prefix unused variables with `_` so we know it's intentionally unused?

Reply:
Added back in something which will assert they match :)